s/shelljs/shelljs_ubi_9.3.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash -e
+# ----------------------------------------------------------------------------
+#
+# Package       : shelljs
+# Version       : master
+# Source repo   : https://github.com/shelljs/shelljs
+# Tested on     : UBI: 9.3
+# Language      : javascript
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Stuti Wali <Stuti.Wali@ibm.com>
+#
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=shelljs
+PACKAGE_VERSION=${1:-master}
```

Comment:
Why we are doing default master. Is same script working for last tag also v0.8.5 also?

Reply:
Latest version was 3 years old, where as master has latest commit 5 months back. So used master over latest version.